src/components/Page.jsx

```diff
@@ -61,6 +39,45 @@ const Page = ({ entry, topic }) => {
           flex: 0 0 16px;
           margin-right: 3px;
         }
+      `}</style>
+    </li>
+  );
+};
+
+const Title = ({ entry }) => {
+  const isJp = entry.country === 'jp';
+  const url = isJp ? entry.url : makeTranslatedUrl(entry.url);
+  const day = dayjs(entry.orig.timestamp).format('MM/DD');
+  const title = entry.ja_translated.title;
+  return (
+    <>
```

Comment:
```suggestion
    <div className="wrap">
```

タイトルとオリジナルページのリンクを `div` で囲みます